[CS2113-T13-2] LOTS

andrewtkh1 opened the pull request:
Large Order Tracking System (LOTS) is a Command Line (CLI) program that enables users to keep track of multiple food orders from a pre-set list of food items from different stores. The program helps users to collate the orders and displays a summary of all the orders along with other information such as each person’s order and special requirements (if any), cost of an individual’s order,  total cost of all the orders and more. As LOTS is a CLI program, this would greatly benefit any user that excels in typing.

brendanlsz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+This is a sequence diagram of the `Parser` class.
+
+![Squence Diagram of Parser](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/Parser%20Sequence%20Diag.jpg)
```
I think this diagram is very informative 👍 However, is the lifeline for the command objects correct? Also, perhaps you could make this diagram a little higher definition? It seems a little blurred to me...

brendanlsz replied:
Is the activation bar of `:Parser` a little too long?

brendanlsz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+The class diagram below is a brief overview of how the `Parser` is used in parsing the user's command.
+
+![Class Diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/ParserClassDiag-Page-1.jpg)
```
Is this a class diagram or object diagram? If it is a class diagram, you may want to remove the `:`.

edlyIrsyad replied:
To add on, for object diagrams, the names of the objects should be underlined

brendanlsz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+Hence, the sequence of which how Menu Command class and Order Command class are very similar. To prevent repeating of Sequence diagrams, a shared diagram will be listed below for the Menu Command Class and the Order Command class.  
+   
+<br>![MenuAndOrdersSequenceDiagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/MenuAndOrdersSequenceDiagram/MenuAndOrdersSequenceDiagram.jpg)    
```
Is there a reason why some sequence diagrams are colored and others are black-and-white? Perhaps it may be better to keep it consistent throughout your DeveloperGuide?

brendanlsz commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+Below is a step by step example on how the `Parser` interacts when a user keys in an input.
+
+Step 1)<br>
```
I think it may be better to change the formatting of these steps to be the same as the format under `Logical Component` section?

brendanlsz reviewed (commented):
Overall, I think your DeveloperGuide is very readable and informative. Just some small formatting issues to take care of. 👍

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+The logical component of the program consists multiple classes. Namely: `Parser`,`Command` &
+the various child class of `Command`.
+The class diagram below is a brief overview of how the `Parser`, `Manger` & the various `Command` class
```
Typo for `Manager`

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+the various child class of `Command`.
+The class diagram below is a brief overview of how the `Parser`, `Manger` & the various `Command` class
+are related to one another.  
+<br>![Logical Component Partial Class Diagram](https://github.com/markuslyq/tp/blob/master/UMLdiagrams/LogicalComponentDiagrams/Logical%20Component%20Diagram-Page-2.jpg?raw=true)
```
Do you think the curved corners of the classes in this class diagram can be changed to sharp corners? Might wanna confirm with the module textbook/prof about this.

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+The class diagram below is a brief overview of how the `Parser` is used in parsing the user's command.
+
+![Class Diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/ParserClassDiag-Page-1.jpg)
```
Should the class diagram here be without the `:`?

edlyIrsyad replied:
Should the Manager class be included in this diagram?

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+
+Explanation on how the parsing is done:
+1. Upon receiving the user's input string from `Duke`, the `Parser` split the user's input into an array of strings.
+2. It then interprets the string and try to match it with one of the known commands.
```
Sorry for being particular about grammar too, but I suggest the same tense should be used. i.e "interprets" and "tries"

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+   if there is no match)
+
+The following sequence diagram depicts how the `Logical` components interact with one another upon receiving the user's input of `"delete 1/2"`.  
+<br>![Logical Component Partial Class Diagram](https://github.com/markuslyq/tp/blob/master/UMLdiagrams/DeleteCommandDiagram/DeleteCommand%20Sequence%20Diagram.jpg?raw=true)
```
In [this diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/ParserClassDiag-Page-1.jpg), it says that `getCommand()` is a **static method** so shouldn't the Parser here include `<<class>>` ?

theeugenechong replied:
I think there is a typo in the diagram:  `new deteteCommand()`

theeugenechong replied:
![image](https://user-images.githubusercontent.com/69380888/139005122-286430d3-9a9c-4a58-9cef-4d02661e503e.png)
Do you think that the return value should be included here, In this case a `DeleteCommand` object?

theeugenechong replied:
![image](https://user-images.githubusercontent.com/69380888/139005240-2170e2af-c22c-4e65-8b52-96519ca8bd02.png)
I think the arrow needs to point to exactly where the activation bar stops

edlyIrsyad replied:
Should there be a delete here?
![image](https://user-images.githubusercontent.com/70310901/139006990-e3fb21b4-1802-42e1-a05a-2742116995bc.png)

edlyIrsyad commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Architecture
```
Should this  'Architecture' section be under the 'Design' section? As seen from the sample AddressBook DG given here https://se-education.org/addressbook-level3/DeveloperGuide.html#proposed-undoredo-feature, the 'Architecture' section should illustrate the high-level design of the program, explaining how the main classes work.

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+### Manager Component
+
+**API** : PeopleManager.java
+<br>![PeopleManagerDiagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/PeopleManagerDiagram/PeopleManagerDiagram.jpg)
```
This diagram is simple and easy to understand! 👍 However, do you think you could explain what the dotted boxes mean? I am guessing they are referring to different packages? Might be confusing for some readers

theeugenechong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,329 @@
+In this case, the `"delete"` command will cause the parser to return a `DeleteCommand` object.
+The image below shows the codes for the switch statement.
+
+![Switch case Code](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/SwitchClassCode.jpg)
```
Do you think it would be better to to this using code rather than a screenshot? For e.g.

```
switch(commandInString)
// all the case statements
```

astralum replied:
I agree with using code, as it makes it easier to edit in case you change the implementation in the future.

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
 # Developer Guide
```
Just wondering, why does your team have a DG for each member in your PR? It makes it confusing to see which DG is the one that's published.

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+## Design
+
+### Logical Component
```
This component is named "Logic" and not "Logical" under "Main components of architecture". Maybe it would be better to maintain consistency of naming.

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+### Logical Component
+
+The logical component of the program consists multiple classes. Namely: `Parser`,`Command` &
```
very small typo: missing whitespace between "`Parser`," and "`Command`"

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+are related to one another.  
+<br>![Logical Component Partial Class Diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/LogicalComponentDiagrams/Logical%20Component%20Diagram-Page-2.jpg)
+
+:information_source: **Note:** Specific command names are represented using a placeholder `'Abc'`, i.e. AddCommand, FindCommand.
```
Nice informational note!

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+Below is a step by step example on how the `Parser` interacts when a user keys in an input.
+
+Step 1)<br>
+Lets assume the user input is `delete 1/2`. `Duke` will then call the method
```
minor typo in "Lets", which should be "Let's"

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+The image below shows the codes for the switch statement.
+
+![Switch case Code](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/SwitchClassCode.jpg)
+
```
If the opt block results in an UnknownCommand being created, and so does the default case in the switch block, is it necessary to have the opt block?

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+Step 1)<br>
+Lets assume the user input is `delete 1/2`. `Duke` will then call the method
+`Parser.getCommand("delete 1/2")`. The method would split the user's input into an array of
+strings along the spaces and store them in the array `listOfInputs`. After which,
```
"split by whitespaces/spaces" might be a better phrase than "along the spaces"

astralum commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+Step 3)<br>
+Next, `Duke` will then use the returned command to call `Command.execute()` which will interact
+with the `Manager Components` of the program who will then remove the 2nd order from the first
```
Did you mean to write "`Manager` component" instead of "`Manager Components`"?
Also, you could split this sentence to avoid having two "which"s in one sentence.

astralum commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,375 @@
+    break;
+```
+The upside of doing would be that there is less code overall.
+However, doing so would cause our code to have a higher amount of coupling and would also
````
Nice explanation!

astralum reviewed (commented):
Good job overall! There are multiple minor grammatical errors, but they do not affect the overall readability of the DG.

xingjie99 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+## Architecture
+
+<br>![Architecture Diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ArchitectureDiagrams/ArchitectureDiagram.drawio.png)
```
Would be good if you are able to have consistent naming throughout the diagrams (eg Logic or Logical) and also use a standardised software to draw the images.

xingjie99 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+This is a sequence diagram of the `Parser` class.
+
+![Squence Diagram of Parser](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/ParserDiagrams/Parser%20Sequence%20Diag.jpg)
```
The sequence diagram for a switch case should be quite clear and shouldn't be a need to include it here. Perhaps also can include the colours that are indicated in architecture make these diagrams even clearer

xingjie99 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+The overview class diagram is listed here to show how the command classes interact on the logical component to the manager component.
+
+The Sequence Diagram below represents the interactions between components when user inputs command `add /n tom /i 1 /q 2`
+<br>![Sequence Diagram](https://raw.githubusercontent.com/thaddyyz/tp/master/UMLdiagrams/EditCommandDiagrams/AddCommandSeqDiagram.png)
+
+The Sequence Diagram below represents the interactions between components when user inputs command `delete 1/2`
+<br>![Sequence Diagram 2](https://raw.githubusercontent.com/thaddyyz/tp/master/UMLdiagrams/EditCommandDiagrams/deleteCommandSeqDiagram.png)
```
the sequence diagrams here are a little too blurry :( the self invocation methods cannot be clearly seen too. Perhaps might want to reconsider using better diagrams and maybe not necessary to put both add and delete (since the sequence diagram for both are quite similar)

xingjie99 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+
+Hence, the sequence of which how Menu Command class and Order Command class are very similar. To prevent repeating of Sequence diagrams, a shared diagram will be listed below for the Menu Command Class and the Order Command class.  
+   
+<br>![MenuAndOrdersSequenceDiagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/MenuAndOrdersSequenceDiagram/MenuAndOrdersSequenceDiagram.jpg)    
```
this diagram is big and clear :) i think that one issue may be that the Ui.printOrdersList() should be pointing to the top of the activation bar instead of the middle of the activation bar.

xingjie99 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,375 @@
+   if there is no match)
+
+The following sequence diagram depicts how the `Logical` components interact with one another upon receiving the user's input of `"delete 1/2"`.  
+<br>![Logical Component Partial Sequence Diagram](https://raw.githubusercontent.com/AY2122S1-CS2113-T13-2/tp/master/UMLdiagrams/DeleteCommandDiagram/DeleteCommand%20Sequence%20Diagram.jpg)
```
This diagram looks very neat and clear, perhaps can use this as an example for your other diagrams (colour codes, size, etc)

thaddyyz committed:
Merge pull request #220 from thaddyyz/master

update EditCommand, Menu

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp into Edits_MA

mohamad-adam8991 committed:
Edited UG to fix PED bugs

mohamad-adam8991 committed:
Additional Edits to UG

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

andrewtkh1 committed:
Fixed DG based on feedback.

markuslyq committed:
Update DeleteCommand Sequence Diagram.drawio

markuslyq committed:
Added DeleteCommand Sequence Diagram.jpg

andrewtkh1 committed:
Merge pull request #222 from andrewtkh1/DGupdate-Drew

DG update drew

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

markuslyq committed:
Made some changes to the diagram of the logical component

markuslyq committed:
Merge pull request #229 from markuslyq/master

Edited some diagrams of the logical component

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

andrewtkh1 committed:
Edit Find UG.

andrewtkh1 committed:
Edit DG to fix indentation.

andrewtkh1 committed:
Edit IO-Redirection test to remove all remaining orders.

Integrated Setup for dev readme into Dev guide.

Edited UML diagram to add background color.

mohamad-adam8991 committed:
Merge pull request #221 from mohamad-adam8991/Edits_MA

Edited UG for PE-D Bugs

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

andrewtkh1 committed:
Merge branch 'master' into PEDBugFix-Drew

andrewtkh1 committed:
Removed .orders.txt

andrewtkh1 committed:
Removed GFM.

andrewtkh1 committed:
Merge pull request #230 from andrewtkh1/PEDBugFix-Drew

PE-D bug fix drew

andrewtkh1 committed:
Fixed Table & restored GFM

Changed UG to try and fix Anchor.

andrewtkh1 committed:
Add empty line.

andrewtkh1 committed:
Merge pull request #231 from andrewtkh1/PEDBugFix-Drew

Fixed Table & restored GFM

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Moved end of Parser Implementation to command implementation.

andrewtkh1 committed:
Merge pull request #232 from andrewtkh1/PEDBugFix-Drew

Ped bug fix drew

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

mohamad-adam8991 committed:
Merge branch 'Edits_MA'

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

WaiKit-nus committed:
Add lifeline of entity for ordersCommand

WaiKit-nus committed:
Edit markdown language.

WaiKit-nus committed:
Merge pull request #233 from WaiKit-nus/master

Change markdown language and uml diagram

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

thaddyyz committed:
Merge pull request #16 from AY2122S1-CS2113-T13-2/master

update

thaddyyz committed:
Update Menu.java

thaddyyz committed:
Update EXPECTED.TXT

thaddyyz committed:
Merge pull request #234 from thaddyyz/master

update menu

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

markuslyq committed:
Added Storage Sequence Diagram

mohamad-adam8991 committed:
Edited DG and UG for formatting

mohamad-adam8991 committed:
Edited DG

andrewtkh1 committed:
Merge pull request #236 from mohamad-adam8991/DG_MA

DG and UG edits

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

mohamad-adam8991 committed:
Minor Edits to UG

mohamad-adam8991 committed:
Minor Edits to set up guide

mohamad-adam8991 committed:
Minor Edits to DG

thaddyyz committed:
Update Ui.java

thaddyyz committed:
update ioredirection

thaddyyz committed:
Update Ui.java

markuslyq committed:
Merge pull request #238 from thaddyyz/master

Update Ui.java

andrewtkh1 committed:
Merge pull request #237 from mohamad-adam8991/DG_MA

Edited Guides

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

thaddyyz committed:
Update menuUDDiagram.png

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
REmoved Orders.txt

markuslyq committed:
Resolved a storage bug.

markuslyq committed:
Update Storage Sequence Diagram

andrewtkh1 committed:
Merge pull request #240 from markuslyq/master

Resolved a storage bug.

andrewtkh1 committed:
Merge pull request #239 from thaddyyz/master

Update menuUGDiagram.png

mohamad-adam8991 committed:
Merge branch 'DG_MA'

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

WaiKit-nus committed:
Update PeopleManagerDiagram

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

WaiKit-nus committed:
Merge pull request #241 from WaiKit-nus/master

Update PeopleDiagram

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Edit Welcome page menu.

markuslyq committed:
Update Storage Sequence Diagram

markuslyq committed:
Added Storage Sequence Diagram.jpg

WaiKit-nus committed:
Merge pull request #242 from andrewtkh1/UpdateWelcomePageImg-Drew

Update welcome page img drew

WaiKit-nus committed:
Edit UserGuide

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

markuslyq committed:
Delete Storage Sequence Diagram.jpg

markuslyq committed:
Delete Storage Sequence Diagram

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
Merge pull request #243 from WaiKit-nus/master

Edit UserGuide

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

markuslyq committed:
Updated Add Command in UG

markuslyq committed:
Added Storage Sequence Diagram.jpg

markuslyq committed:
Update Storage Sequence Diagram.jpg

markuslyq committed:
Update Storage.java

markuslyq committed:
Update Person.java

andrewtkh1 committed:
Merge pull request #244 from markuslyq/master

Added a storage sequence diagram.

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

markuslyq committed:
Added a more specific error message for delete and edit command

markuslyq committed:
Merge pull request #246 from markuslyq/master

Added a more specific error message for delete and edit command

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

mohamad-adam8991 committed:
Added header comment to AddCommandTest

mohamad-adam8991 committed:
Refactored parts of the add command class to apply SLAP

thaddyyz committed:
Merge pull request #17 from AY2122S1-CS2113-T13-2/master

update

thaddyyz committed:
UpdateUG

thaddyyz committed:
Create thaddyyz.md

andrewtkh1 committed:
Merge remote-tracking branch 'Upstream/master'

markuslyq committed:
Create markuslyq.md

andrewtkh1 committed:
Final edts to IO-Redirect.

andrewtkh1 committed:
Merge pull request #248 from andrewtkh1/IORedirect-FinalCheck

IO redirect final check

andrewtkh1 committed:
Merge remote-tracking branch 'Upstream/master'

mohamad-adam8991 committed:
Refactored portions of the AddCommand to implement SLAP

mohamad-adam8991 committed:
Edited UG to add storage component

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Update markuslyq.md

mohamad-adam8991 committed:
Edited DG

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp into BugFixes_MA

markuslyq committed:
Merge branch 'master' of https://github.com/markuslyq/tp

mohamad-adam8991 committed:
Edited format of DG

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Merge pull request #250 from markuslyq/master

Markus's PPP

markuslyq committed:
Update UG_DeleteCommand_Example.png

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

mohamad-adam8991 committed:
Updates to storage DG

mohamad-adam8991 committed:
Shifted storage

markuslyq committed:
Merge pull request #251 from markuslyq/master

Update UG_DeleteCommand_Example.png

mohamad-adam8991 committed:
Merge pull request #249 from mohamad-adam8991/BugFixes_MA

Edited DG, UG and refactored AddCommand

markuslyq committed:
Update Storage Sequence Diagram

markuslyq committed:
Added Storage Sequence Diagram.jpg

markuslyq committed:
Update Storage Sequence Diagram

markuslyq committed:
Added Storage Sequence Diagram.jpg

markuslyq committed:
Update Storage Sequence Diagram

markuslyq committed:
Update DeleteCommand Sequence Diagram.drawio

markuslyq committed:
Added DeleteCommand Sequence Diagram.jpg

markuslyq committed:
Update Logical Component Diagram.drawio

markuslyq committed:
Added Logical Component Diagram.jpg

mohamad-adam8991 committed:
Update architecture diagram

andrewtkh1 committed:
Add PPP for drew.

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into PPP-Drew

andrewtkh1 committed:
Added Test cases.

markuslyq committed:
Edited the storage implementation in the DG

andrewtkh1 committed:
Edit UG according to new Menu.

markuslyq committed:
Merge pull request #253 from markuslyq/master

Edited the storage implementation in the DG

andrewtkh1 committed:
Merge pull request #252 from andrewtkh1/PPP-Drew

PPP drew

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp into BugFixes_MA

mohamad-adam8991 committed:
Edited DG

mohamad-adam8991 committed:
Merge pull request #254 from mohamad-adam8991/BugFixes_MA

Edited DG

mohamad-adam8991 committed:
Merge branch 'BugFixes_MA'

markuslyq committed:
Update markuslyq.md

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Edits to DG.

thaddyyz committed:
Merge pull request #247 from thaddyyz/master

update UG and PPP

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
Merge branch 'master' into MinorBugFix-Drew

andrewtkh1 committed:
Fix for UG Caution.

andrewtkh1 committed:
Add Space.

mohamad-adam8991 committed:
Edited UG image

andrewtkh1 committed:
Merge pull request #255 from andrewtkh1/MinorBugFix-Drew

Minor bug fix in DG Drew

markuslyq committed:
Add Space

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp into BugFixes_MA

markuslyq committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

mohamad-adam8991 committed:
Minor edits to UG

markuslyq committed:
Update Storage Sequence Diagram

markuslyq committed:
Added Storage Sequence Diagram.jpg

mohamad-adam8991 committed:
Minor Edits to UG

markuslyq committed:
Merge pull request #257 from markuslyq/master

Shifted the return line in storage sequence diagram down

mohamad-adam8991 committed:
Merge pull request #256 from mohamad-adam8991/BugFixes_MA

Edited UG list image

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp into BugFixes_MA

mohamad-adam8991 committed:
Minor Edits

mohamad-adam8991 committed:
Minor Edits

mohamad-adam8991 committed:
Merge pull request #258 from mohamad-adam8991/BugFixes_MA

Minor Fixes

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

thaddyyz committed:
Merge pull request #18 from AY2122S1-CS2113-T13-2/master

update

andrewtkh1 committed:
Fix for PPP hyperlink.

WaiKit-nus committed:
Merge pull request #259 from andrewtkh1/master

Fix PPP hyperlink.

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

andrewtkh1 committed:
try Fix

WaiKit-nus committed:
Merge pull request #260 from andrewtkh1/FixPPPLinks

Fix ppp links

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

thaddyyz committed:
Merge pull request #19 from AY2122S1-CS2113-T13-2/master

update

thaddyyz committed:
Update editUGDiagram.png

thaddyyz committed:
Merge pull request #261 from thaddyyz/master

update UG image

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

markuslyq committed:
Corrected a minor preposition error.

WaiKit-nus committed:
Resolve issues.

WaiKit-nus committed:
Merge pull request #263 from WaiKit-nus/ResolveIssues

Resolve issues

markuslyq committed:
Merge pull request #262 from markuslyq/master

Corrected a minor preposition error.

andrewtkh1 committed:
Merge remote-tracking branch 'upstream/master'

markuslyq committed:
Update markuslyq.md

markuslyq committed:
Made some minor changes to UG.

markuslyq committed:
Made some minor changes to DG

andrewtkh1 committed:
Removed headers

andrewtkh1 committed:
Merge pull request #265 from andrewtkh1/UpdatePPP-Drew

Update ppp drew

andrewtkh1 committed:
Removed header.

andrewtkh1 committed:
Compressed PPP.

andrewtkh1 committed:
Merge pull request #266 from andrewtkh1/UpdatePPP-Drew

Removed header. PPP

andrewtkh1 committed:
Minor Edit.

markuslyq committed:
Merge pull request #264 from markuslyq/master

Update markuslyq.md

andrewtkh1 committed:
Merge pull request #267 from andrewtkh1/UpdatePPP-Drew

Minor Edit.

mohamad-adam8991 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113-T13-2/tp

mohamad-adam8991 committed:
Added Adam's PPP

mohamad-adam8991 committed:
Merge pull request #268 from mohamad-adam8991/PPP_MA

Added Adam's PPP

WaiKit-nus committed:
Add WaiKit PPP

WaiKit-nus committed:
Merge pull request #269 from WaiKit-nus/WKPPP

Add WaiKit PPP

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

markuslyq committed:
Format changes to PPP

markuslyq committed:
Merge pull request #270 from markuslyq/master

Format changes to PPP

markuslyq committed:
Updated a link for PPP

markuslyq committed:
Made a minor change in the command summary of the UG

markuslyq committed:
Added a few details to the add feature in the UG

markuslyq committed:
Change a phrasing in the architecture section.

markuslyq committed:
Merge pull request #271 from markuslyq/master

Made some minor changes to DG and UG.

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

WaiKit-nus committed:
Add header comments for AddCommandTest

WaiKit-nus committed:
Checkstyle Clear for Add Command Test

andrewtkh1 committed:
Merge pull request #272 from WaiKit-nus/master

Add Header comment for AddCommandTest

WaiKit-nus committed:
Refactor naming for PPP

andrewtkh1 committed:
Merge pull request #273 from WaiKit-nus/master

Refactor naming for PPP

andrewtkh1 committed:
Add Javadocs for test case. Removed img.png file which was not used. Edit Waikit's PPP format.

WaiKit-nus committed:
Merge pull request #274 from andrewtkh1/TestingComments-Drew

Add Javadocs for test case. Removed img.png file which was not used. …

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

WaiKit-nus committed:
Reduce WKPPP

markuslyq committed:
Added JavaDocs for deleteCommandTest

WaiKit-nus committed:
Merge pull request #275 from markuslyq/master

Added JavaDocs for deleteCommandTest

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

WaiKit-nus committed:
Merge pull request #276 from WaiKit-nus/master

Edit WKPPP

WaiKit-nus committed:
Merge remote-tracking branch 'upstream/master'

WaiKit-nus committed:
Edit spacing for WKPPP

WaiKit-nus committed:
Merge pull request #277 from WaiKit-nus/master

Change spacing for WKPPP

andrewtkh1 committed:
Added Value Proposition & Target User.

andrewtkh1 committed:
Corrected "dinners" to "diners"

andrewtkh1 committed:
Merge pull request #278 from andrewtkh1/EditUG-Drew

Added Value Proposition & Target User.
